.env.example

```diff
@@ -8,6 +8,9 @@ PORT=5000
 
 DEBUG=false
 
+# Baseurl for creating urls via the backend
+LNBITS_BASEURL="http://0.0.0.0/"
```

Comment:
```suggestion
LNBITS_BASEURL="http://127.0.0.1/"
```

Reply:
Some OS'es can't connect to `0.0.0.0`, it's only safe to use it for _listening_ on an IP (not connecting).

Reply:
removed it from .env.template for now, as we talked